k8s.gcr.io/images/k8s-staging-kustomize/images.yaml

```diff
@@ -29,3 +29,4 @@
     "sha256:33870c75c8ebfea09226a4a7d9e955938780516f3242afff61bf296a6b54b443": ["v4.5.3"]
     "sha256:75ccf9a833ca288c17b20ef61052dc8aefd6b99790585cef08734686e8062ebb": ["v4.5.4"]
     "sha256:05e12a3e5085902d3d1b99a0ce74d7d9cead0d8a0343df7f306db7b1616a27b5": ["v4.5.5"]
+    "sha256:b8138fa08c202dd88474d7d8e7752265402f1bb800c94d80d5445ac9be1dcee4": ["v4.5.7"]
```

Comment:
This should be 8d02f0774cde076e23a6d7880a85f56d6cab48cbb4dfe4e3f90e3618a0ca606e ([here](https://console.cloud.google.com/gcr/images/k8s-staging-kustomize/global/kustomize@sha256:8d02f0774cde076e23a6d7880a85f56d6cab48cbb4dfe4e3f90e3618a0ca606e/details)), i.e. the one with the bare 4.5.7 tag. The ones with sha suffixes are built off master.

Reply:
🤦

Reply:
fixed!